src/xkbcomp/rules.c

```diff
@@ -988,8 +988,45 @@ expand_rmlvo_in_kccgst_value(struct matcher *m, struct scanner *s,
 }
 
 /*
- * This function performs %-expansion on @value (see overview above),
- * and appends the result to @to.
+ * This function performs :all replacement on @value (see overview above),
+ * and appends the result to @expanded.
+ */
+static bool
```

Comment:
Since the `bool` return value is not used, would remove it.

Reply:
Done